decidim-api/decidim-api.gemspec

```diff
@@ -33,6 +33,7 @@ Gem::Specification.new do |s|
   end
 
   s.add_dependency "decidim-core", Decidim::Api.version
+  s.add_dependency "devise-jwt"
```

Comment:
I think we need to lock this to the latest minor version in order to avoid issues during upgrades.